package protoFile.guild.AppliOrUnsubscribe;
message AppliOrUnsubscribeRequest{
     required int32 id = 1;//角色id
     required int32 operType = 2;//操作类型 0 申请进入 1 取消申请
     required int32 corpsId = 3;//军团ID
}
message AppliOrUnsubscribeResponse{
     required bool result = 1;
     optional string message = 2;
}